components/TripDetailView.js

```diff
@@ -1,17 +1,12 @@
 import { styled } from "styled-components";
 import PrimaryLink from "./PrimaryLink";
 
-export default function TravelOverview({
-  trip,
-  handleDelete,
-  handleEdit,
-  tripId,
-  todos,
-}) {
+export default function TravelOverview({ trip, handleDelete, tripId, todos }) {
   /* page 2 detail view*/
```

Comment:
The comment is a little confusing as in the next line you handle your model and not the "page 2 detail view". Do you really need the comment to understand the purpose of the page? If so I would move the code up a few lines.